R/model.R

```diff
@@ -417,6 +417,10 @@ compile <- function(quiet = TRUE,
   if (length(self$stan_file()) == 0) {
     stop("'$compile()' cannot be used because the 'CmdStanModel' was not created with a Stan file.", call. = FALSE)
   }
+  if (!is.null(cpp_options[["stan_threads"]]) && !is.logical(cpp_options[["stan_threads"]]) ||
+      !is.null(private$precompile_cpp_options_[["stan_threads"]]) && !is.logical(private$precompile_cpp_options_[["stan_threads"]])) {
```

Comment:
I just realized that this is going to prevent `stan_threads = "true"` or `stan_threads = "TRUE"` and require the user to use `stan_threads = TRUE` only. That's ok with me, but if we want to allow the strings too I guess we'd need to make this check more complicated. What do you think? We can also just skip this warning if you prefer.

Reply:
Oh right. Lets just skip it then. Will fix a bit later.

Reply:
Ok sounds good